identity_core/tests/authentication.rs

```diff
@@ -0,0 +1,546 @@
+use identity_core::{
+    document::DIDDocument,
+    utils::{Authentication, Context, PublicKey, Subject},
+};
+
+use std::str::FromStr;
+
+#[test]
+fn test_auth() {
+    let json_str = r#"
```

Comment:
some of these large strings can go in files and be added through `include_str!`

Reply:
Already done, in the latest version.